content/docs/components/introduction.md

```diff
@@ -57,18 +57,33 @@ The component props are passed as the second parameter using the `@component` ta
 
 See also: [Props reference](./props.md)
 
+:::note
+
+In Edge.js, when you pass attributes such as class, they can sometimes conflict with the reserved keywords or syntax. You can rename the class attribute to something else, like `btnClass`, and use that in both the component definition and when rendering the component.
+
+
+:::
+
+```edge
+// title: views/components/button.edge
+<button type="{{ type || 'submit' }}" class="{{ btnClass }}">
+    {{ text }}
+</button>
+```
+
+
```

Comment:
Not sure I really want to add this, maybe a better way to use the `class` prop is via the `$props.get('class')` call

Reply:
"Unexpected token" error with and not separately closed:

@component('components/button', {
  type: 'submit',
  class: 'btn btn-large',
  text: 'Login'
})
`

To avoid these errors, the documentation should clarify the correct approach, since 'class' is a reserved word. For example:
`
<button type="{{ type || 'submit' }}" class="{{ $props.get('class') }}">
  {{ text }}
</button>
`

This can now be used in templates as follows, with the option to use self-closing syntax (@!component) if needed:
`
@component('components/button', {
  type: 'submit',
  class: 'btn btn-large',
  text: 'Login'
})
@end
`